modin/pandas/test/dataframe/test_default.py

```diff
@@ -214,11 +213,39 @@ def test_combine_first():
     df_equals(modin_df1.combine_first(modin_df2), pandas_df1.combine_first(pandas_df2))
 
 
-def test_cov():
-    data = test_data_values[0]
-    modin_result = pd.DataFrame(data).cov()
-    pandas_result = pandas.DataFrame(data).cov()
-    df_equals(modin_result, pandas_result)
+@pytest.mark.parametrize("min_periods", [3, 5, 7])
```

Comment:
What is expected difference between results of `corr` using `min_periods` from this list? I think, probably, using default value and one other is enough in this case. What do you think? 

Also, pay attention on `ddof` and `min_periods` parameters for `cov` function test.

Reply:
`min_periods` affects result but not significantly. Modin result may differ from pandas result in the last units. Difference may be about 10^-17 or 10^-18. I think two or three values differ from default are enough.